gpt_neox/datasets.py

```diff
@@ -7,8 +7,98 @@
 import re
 import logging
 from itertools import cycle
+import simdjson as json
+import linecache
+import numpy as np
+from timeit import default_timer as timer
+
+"""
+Dataset that gets sequences from a set of sharded jsonl files
+"""
+class JsonShardedDataset(Dataset):
```

Comment:
It would be helpful to have the doc-comment attached to the class. So do:

```
class JsonShardedDataset(Dataset):
    """
    Dataset that gets sequences from a set of sharded jsonl files
    """
```

instead of it dangling above.